bower.json

```diff
@@ -45,6 +45,7 @@
     "knockout": "3.3.0",
     "knockout-validation": "2.0.3",
     "underscore": "1.8.3",
+    "underscore.string": "3.3.4",
```

Comment:
Are you using this for anything other than `capitalize`? I just skimmed the PR. We have a [local version](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/export/static/export/js/utils.js#L34).

Reply:
🤔  not sure if that's the most intuitive place for it lol... exports?

Reply:
Exports is the only place it was being used. Would make sense to move it someplace like main.js and use it here.

Reply:
Also, this doesn't do quite the same thing. This assumes that a word is passed in, but `s.capitalize` handles multiple words separated by a space. That case doesn't really apply to the report builder analytics, so I moved capitalize into `main.js` and removed `underscore.string` anyway. But the `underscore.string` library is way more powerful. I would prefer to have that available, but maybe later.

Reply:
Yeah, let's talk about it when there's a stronger definite need for it. It might make sense to replace underscore with lodash at that point rather than adding underscore.string.